### DISCUSSION TITLE
[1.3] Importing the state_vector_exercise results in an error

### DISCUSSION THREAD

**Discussion by oevers** [qiskit-community/qiskit-textbook#1366] [Q&A]
### Describe the bug

Importing the state_vector_exercise results in an error

> # Run the code in this cell to interact with the widget
> from qiskit_textbook.widgets import state_vector_exercise
> state_vector_exercise(target=1/3)

Results in the following:

> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> <ipython-input-17-2fbbdd615fc2> in <module>
>       1 # Run the code in this cell to interact with the widget
> ----> 2 from qiskit_textbook.widgets import state_vector_exercise
>       3 state_vector_exercise(target=1/3)
> 
> ~/opt/anaconda3/envs/qiskit_env/lib/python3.8/site-packages/qiskit_textbook/widgets/__init__.py in <module>
>       5 from qiskit.visualization import plot_bloch_vector
>       6 from numpy import sqrt, cos, sin, pi
> ----> 7 import numexpr
>       8 import re
>       9 
> 
> ModuleNotFoundError: No module named 'numexpr'

### To Reproduce

Run the textbook in anaconda in an environment with qiskit and the textbook installed as described here https://qiskit.org/textbook/ch-prerequisites/setting-the-environment.html

### Expected behavior

widget should come up without errors

### Version info

<h3>Version Information</h3>

Qiskit Software | Version
-- | --
qiskit-terra | 0.18.3
qiskit-aer | 0.9.1
qiskit-ignis | 0.6.0
qiskit-ibmq-provider | 0.17.0
qiskit | 0.31.0
qiskit-nature | 0.2.2
qiskit-finance | 0.2.1
qiskit-optimization | 0.2.3
qiskit-machine-learning | 0.2.1

Python | 3.8.12 (default, Oct 12 2021, 06:23:56)  [Clang 10.0.0 ]
OS | Darwin
CPUs | 6
Memory (Gb) | 16.0
Sat Dec 18 21:20:42 2021 CET

**Comment by oevers** [comment]
It turned out that numexpr wasn't installed in my environment
once I did a `pip install numexpr` in a cell before calling the widget, things started working.
The question is now why is numexpr not a prereq for the qiskit_textbook so that it gets installed as a prereq by pip?

**Reply by paniash** [reply]
How did you install the textbook? If you notice in `requirements.txt`, the `numexpr` package exists, so it should have been installed as a dependency.

**Reply by oevers** [reply]
I believe I did a standard pip install a few months ago, so will have been somewhat backlevel compared to what is current now. Has the requirement.txt changed? Could it have been bypassed?

**Comment by paniash** [comment]
Migrated to a discussion since the issue has been rectified.
